app/models.py

```diff
@@ -96,3 +98,132 @@ def facet_plural(facet: str):
         facet_plural = facet
 
     return facet_plural
+
+
+# --------------------------------------------
+# Response model class for the knowledge panels
+# --------------------------------------------
+
+
+class TextFacet(BaseModel):
+    """Base facet containing text"""
+
+    element_type: str
+    text_element: Optional[str] = Field(
+        default=None,
+        description="Html value with desciption of the item",
+    )
+
+
+class HungerGameElement(TextFacet):
+    id: int
+
+
+class DataQualityAndLastEditsItem(BaseModel):
+    """
+    contains link and text elements
+    """
+
+    title: str
+    subtitle: Optional[str] = Field(
+        default=None,
+        description="Short description of different facet and value in which it computes the data. ",  # noqa: E501
```

Comment:
Maybe instead of using noqa, you should use triple quotes ?

```suggestion
        description="""
            Short description of different facet and value 
            in which it computes the data. """,
```